tox.ini

```diff
@@ -39,7 +40,7 @@ basepython =
 basepython=python
 deps=flake8==3.5.0
 commands=
-    flake8 {toxinidir}/evm
+    flake8 {toxinidir}/evm --exclude="{toxinidir}/evm/chains/sharding/contracts/"
```

Comment:
I'm assuming this is to avoid linting the viper files.  I would suggest changing this to `--exclude="*.v.py"`

Reply:
Will do. Thank you for the reminder